index.js

```diff
@@ -1 +1,30 @@
 // Your code here
+  function showTime(){
+      let date = new Date();
+      let h = date.getHours();
+      let m = date.getMinutes();
+      let s = date.getSeconds(); 
+      let session = "AM";
+
+      if(h == 0){
+          h = 12;
+      }
+
+      if(h > 12){
+          h = h - 12;
+          session = "PM";
+      }
+
+      h = (h < 10) ? "0" + h : h;
+      m = (m < 10) ? "0" + m : m;
+      s = (s < 10) ? "0" + s : s;
+
+      let time = h + ":" + m + ":" + s + " " + session;
+      document.getElementById("clock").innerText = time;
+      document.getElementById("clock").textContent = time;
+
+      setInterval(showTime, 1000);
+
+  }
+
+  showTime();
```

Comment:
This should be inside `$(document).ready` otherwise it's not guaranteed to be executed after the document loads.